zerver/webhooks/gocd/doc.md

```diff
@@ -1,24 +1,39 @@
-Zulip supports integration with GoCD and can notify you of
-your build statuses.
+# Zulip GoCD integration
+
+Get GoCD notifications in Zulip!
+
+!!! warn ""
+
+    **Note**: We currently only support version **v0.0.6** of Sentry's GoCD notification
+    plugin.
```

Comment:
I think it might make sense to  just roll this into the "Download the far file ..." step? They just need to make sure they have version v0.0.6 or higher in that step, if it's already installed (but presumably they don't need to check anything if it's already new).